andela/coding-geeks-ah-frontend
# #167484541 - Setup e2e tests for the application

#### What does this PR do?
- Sets up end to end tests for the application

#### Description of Task to be done?
- Install cypress
- User cypress integration to test the UI

#### How should this be manually tested?
1. Clone this branch.
2. Do `npm install`
3. Do `npm start` and you should see the landing page.
4. Add this virtual environment `SERVER_URL=http://localhost:3000` to the `.env` file
5. In another terminal window run `npm run e2e:test` to run the end to end test

#### Screenshots
<img width="1197" alt="Screen Shot 2019-09-25 at 11 29 32" src="https://user-images.githubusercontent.com/50101879/65589943-3473dc80-df8a-11e9-81d8-55efcb8482f1.png">
